### PR TITLE
Note App

### PR DESCRIPTION
**Pull request by Odile078**
# P.R. Description
Implementation of Note app.
## Completed Tasks
- [x] Add page design
- [x] Add create, update and delete functionality
## Live link
[link](https://deploy-preview-3--magenta-moonbeam-158f96.netlify.app/)
## Testing
Run the command below to open the landing page on a local server for testing
- npm install
- npm run dev

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App'
-import './index.css'
+import React from "react";
```
Comment (mutesialine) [inline src/main.jsx:1]: Better to remove this import of react since the new version of react can handle it by itself

File: src/main.jsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+import ReactDOM from "react-dom";
+import App from "./App";
+import "./index.css";
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
```
Comment (mutesialine) [inline src/main.jsx:9]: Also  you can change this `<React.strictMode>` to  named export .   [here](https://medium.com/@etherealm/named-export-vs-default-export-in-es6-affb483a0910) the link can guide you

**Review by mutesialine — CHANGES_REQUESTED** [review]
Great Job @Odile078  , Only a few things to change for the code to look more better.

**Review by mutesialine — APPROVED** [review]

**Comment by Odile078** [comment]
This pr was created on opened on the wrong branch, which is why it's being closed. It should have been from dev-note-app to note-app instead of dev-note-app to main branch

**Pull request closed without merging**